wsdlgo/encoder.go

```diff
@@ -590,14 +602,41 @@ func (ge *goEncoder) writeSOAPFunc(w io.Writer, d *wsdl.Definitions, op *wsdl.Op
 	if _, exists := ge.soapOps[op.Name]; !exists {
 		return false
 	}
-	// TODO: handle other input params: e.g. SOAP headers.
-	if len(in) < 1 || len(out) != 2 {
-		return false
-	}
+
+	// Do we need to wrap into a operation element?
+	rpcStyle := d.Binding.SOAPBinding.Style == "rpc"
+
 	ge.needsStdPkg["encoding/xml"] = true
-	ge.needsExtPkg["github.com/fiorix/wsdl2go/soap"] = true
-	in[0] = renameParam(in[0], "α")
-	out[0] = renameParam(out[0], "β")
+	ge.needsExtPkg["github.com/Kernle32DLL/wsdl2go/soap"] = true
```

Comment:
This will generate code that does not compile when people use the original repo.